lib/engine/game/g_1849/game.rb

```diff
@@ -83,43 +91,53 @@ class Game < Game::Base
             train_limit: 2,
             tiles: %i[yellow green brown],
             operating_rounds: 3,
-            status: ['gray_uses_black'],
+            status: %w[gray_uses_black e_tokens],
           },
           {
             name: '16H',
             on: '16H',
             train_limit: 2,
             tiles: %i[yellow green brown],
             operating_rounds: 3,
-            status: %w[gray_uses_black blue_zone],
+            status: %w[gray_uses_black blue_zone e_tokens],
           },
         ].freeze
 
-        TRAINS = [{ name: '4H', num: 4, distance: 4, price: 100, rusts_on: '8H' },
-                  {
-                    name: '6H',
-                    distance: 6,
-                    price: 200,
-                    rusts_on: '10H',
-                    events: [{ 'type' => 'green_par' }],
-                  },
-                  { name: '8H', distance: 8, price: 350, rusts_on: '16H' },
-                  {
-                    name: '10H',
-                    num: 2,
-                    distance: 10,
-                    price: 550,
-                    events: [{ 'type' => 'brown_par' }],
-                  },
-                  {
-                    name: '12H',
-                    num: 1,
-                    distance: 12,
-                    price: 800,
-                    events: [{ 'type' => 'close_companies' }, { 'type' => 'earthquake' }],
-                  },
-                  { name: '16H', distance: 16, price: 1100 },
-                  { name: 'R6H', num: 2, available_on: '16H', distance: 6, price: 350 }].freeze
+        def game_trains
+          train_list = [
```

Comment:
leave the trains as they were and call `super` here to get the trains before modifying.